src/public/modules/forms/admin/controllers/view-responses.client.controller.js

```diff
@@ -153,20 +170,97 @@ function ViewResponsesController(
         })
       })
       .catch(function (error) {
-        try {
-          const { errorCount, errorMessage } = JSON.parse(error.message)
-          Toastr.error(
-            errorMessage ||
-              `Error downloading. ${errorCount} response(s) could not be decrypted. Please try again later.`,
-            { timeOut: 3000 },
+        const isCircuitTripped = CircuitBreaker.isOurError(error)
+        if (!isCircuitTripped) {
+          try {
+            const { errorCount, errorMessage } = JSON.parse(error.message)
+            Toastr.error(
+              errorMessage ||
+                `Error downloading. ${errorCount} response(s) could not be decrypted. Please try again later.`,
+              { timeOut: 3000 },
+            )
+          } catch (error) {
+            Toastr.error(`Error downloading. Please try again later.`, {
+              // Should still show error message to let user know download has failed
+              timeOut: 3000,
+            })
+            console.error(
+              'Unknown download encrypted responses error:\t',
+              error,
+            )
+          }
+        } else {
+          // TODO: During soft launch, downloads will proceed even in breaker open state
+          // When circuit breaker is hard launched, replace this code chunk with toastr informing user to try again in XX minutes
+          // Use else here instead of fallback because fallback will trigger once circuit trips, even on the first failed call
```

Comment:
would it be possible to use `fallback` but use `awaitDownload` to determine whether or not to retry the download?

Reply:
awaitDownload was intended to prevent premature setting of vm.csvDownloading = false everytime the function is called, so would not have been suitable for this use case. In any case, I've removed it and replace with promise (explained above)